.github/workflows/ci.yml

```diff
@@ -8,57 +8,57 @@ on:
 
 jobs:
   chore:
-    name: 'Testing chores'
+    name: "Testing chores"
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - run: |
           git fetch --no-tags --unshallow origin HEAD main
 
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
-          node-version: 14
```

Comment:
Is something broken in the latest Node 14 that requires pinning?

Reply:
Something is broken with the all Node.js 14 versions but the latest (https://github.com/yarnpkg/berry/issues/4157).
The GHA runner is still picking up 14.19.1 (because their cache is outdated), pinning the version ensures the correct version is installed. This can be reverted in a few weeks I hope.

Reply:
Sounds good, thanks 👍